MedicationScheduler/AuthApp/views.py

```diff
@@ -0,0 +1,51 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login, authenticate, logout
+from AuthApp.forms import RegistrationForm
+from django.contrib.auth.forms import AuthenticationForm
+
+# Create your views here.
+
+def registration_view(request):
+    context = {}
+    if request.POST:
+        form = RegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
```

Comment:
Just a question. Should we create an empty profile for a user when they are registered? We can then fill in the information later if the user provides it on the account page. Alternatively, we could wait to make the profile until they fill in account information. I'm not sure which is best.

Reply:
^ same, I don't know what a better approach is. If I were a user, I would be more comfortable providing my info after creating an account.

Reply:
I was thinking more from the backend should we have a profile associated with every user (even if empty) and fill them when a user chooses or only create a profile for a user when they add profile information.

Reply:
@bew5294 yep, make sense. I'll add it later, okay?